print/Cargo.toml

```diff
@@ -20,3 +20,5 @@ defmt-decoder = { version = "=0.3.11", path = "../decoder", features = [
 ] }
 log = "0.4"
 tokio = { version = "1.38", features = ["full"] }
+notify = "6.1.1"
```

Comment:
Should we disable crossbeam? See https://docs.rs/notify/latest/notify/index.html#crossbeam-channel--tokio

Reply:
Seeing as nobody could reliably confirm the issue and since std has been using the crossbeam implementation since 1.67 anyway I don't see a reason to do so.
See also the last comment on the issue: https://github.com/notify-rs/notify/issues/380#issuecomment-2186929524